src/Logic/operate.js

```diff
@@ -0,0 +1,30 @@
+const Big = require('big.js');
+
+
+export default function operate(numberOne, numberTwo, operation) {
+  const operationssrt = ['+/-', '%', '÷', 'X', '-', '+'];
+  let x; let y; let
+    z;
+  switch (operation) {
+    case operationssrt[0]:
+      return new Big(numberOne).mul(-1).toFixed();
+    case operationssrt[1]:
+      return new Big(numberOne).mod(new Big(numberTwo)).toFixed();
```

Comment:
case: %  -> we are not returning the reminder, we need to get the percentage

could be something like this: 
numA.times(numB).times(100)

Reply:
i will do it but if is not like that you ll pay for it.